client/src/pages/Faq.js

```diff
@@ -89,12 +89,12 @@ const Faq = () => {
               How can I post an offer or request for help on FightPandemics?
             </summary>
             <p>
-              As a registered user you have unlimited access to view and create
+              When you join the community, you will have unlimited access to view and create
               posts and comments on FightPandemics platform.
             </p>
             <p>
-              To sign up as a user click the menu button and select the
-              "Register/Log in" option. There you will be able to sign up with
+              To join now as a user click the menu button and select the
+              "Join Now/Sign In" option. There you will be able to join now with
```

Comment:
Maybe change here the wording to just say `... able to join with ...` instead of `... able to join now with ...` What do you think @doheekim93 ?

Reply:
Yea, agree with @idjevm "able to join with" flows better!

Reply:
updated